CubeLineMoment.py

```diff
@@ -580,11 +627,9 @@ def cubelinemoment_multiline(cube, peak_velocity, centroid_map, max_map,
             hdu = mom.hdu
             hdu.header.update(cube.beam.to_header_keywords())
             hdu.header['OBJECT'] = cube.header['OBJECT']
-            #print(noisemap.std())
             #print("noisemapbright peak = {0}".format(np.nanmax(noisemapbright)))
-            print(np.nanstd(noisemapbright))
             hdu.header['MASKLEV0'] = (np.nanstd(noisemapbright).value,'Spatial masking stdev (noisemapbright)')
-            hdu.header['MASKLEV1'] = (noisemap.std().value,'Spectral masking stdev (noisemap)')
+            hdu.header['MASKLEV1'] = (np.nanstd(noisemap).std().value,'Spectral masking stdev (noisemap)')
```

Comment:
error here: the stddev of the stddev is probably nan
```suggestion
            hdu.header['MASKLEV1'] = (np.nanstd(noisemap).value,'Spectral masking stdev (noisemap)')
```